charts/victoria-metrics-distributed/values.yaml.gotmpl

```diff
@@ -0,0 +1,232 @@
+vmdistributed:
+  fullnameOverride: "vm-distributed"
+
+  common:
+    vmauth:
+      spec:
+        port: "8427"
+        resources:
+          # rounded defaults from Operator
+          requests:
+            memory: "100Mi"
+            cpu: "0.1"
+          limits:
+            memory: "300Mi"
+            cpu: "0.5"
+        podMetadata:
+          labels:
+            # common label for all components of the release (except vmagent extra)
+            # we use these labels in network policies. We do not use official convention
+            # "app.kubernetes.io/part-of" as VM Operator does not support override of some of
+            # official labels (e.g. app.kubernetes.io/name). So we use a shorter version.
+            # See https://github.com/VictoriaMetrics/helm-charts/issues/2545#issuecomment-3496376407
+            part-of: {{ .Release.Name }}
+    vmagent:
+      # https://docs.victoriametrics.com/operator/api/#vmagentspec
+      spec:
+        port: &vmAgentPort "8429"
+        resources: &vmAgentResources
+          # rounded defaults from Operator
+          requests:
+            memory: "256Mi"
+            cpu: "0.1"
+          limits:
+            memory: "512Mi"
+            cpu: "0.5"
+        podMetadata:
+          labels:
+            part-of: {{ .Release.Name }}
+    vmsingle:
+      # https://docs.victoriametrics.com/operator/api/#vmsinglespec
+      spec:
+        port: "8428"
+        removePvcAfterDelete: false
+        storage:
+          resources:
+            requests:
+              storage: 50Gi
+        resources:
+          # rounded defaults from Operator
+          requests:
+            memory: "500Mi"
+            cpu: "0.1"
+          limits:
+            memory: "1.5Gi"
+            cpu: "1.5"
+        podMetadata:
+          labels:
+            app: vmsingle
+            part-of: {{ .Release.Name }}
+
+  zoneTpl:
+    common:
+      spec:
+        nodeSelector:
+          ops: "true"
+        useStrictSecurity: true
+    vmcluster:
+      enabled: false
+    vmsingle:
+      enabled: true
+      spec:
+        # Ideally it should be defined in common spec. But in original chart
+        # values it is defined in zoneTpl. So, common spec is overriden.
+        retentionPeriod: "180d" # ~3 months
```

Comment:
minor enhancement: this could/should be an env-var, it will vary between master/stag/prod (become longer)